docs/man/xrdp-chansrv.8.in

```diff
@@ -28,16 +28,32 @@ Remote Applications Integrated Locally
 Dynamic Virtual Channel
 .RS
 
+.SH ENVIRONMENT
+.TP
+.I CHANSRV_LOG_LEVEL
+Logging level for chansrv from 0 (minimal) to 5 (extensive). Defaults to 3.
```

Comment:
I usually specify the log level by name. Never tried numeric values, it will work though.